zuul.d/projects.yaml

```diff
@@ -261,3 +259,10 @@
             branches: master
         - flink-end-to-end-test-part3:
             branches: master
+
+- project:
```

Comment:
I think it's better to move this block to the related Periodic. periodic-0 task should be under periodic-0/12 "####################### periodic jobs on 00:00/12:00 ##########################"